### PR TITLE
Fix footer and changed the text in Account Detail

### PR DESCRIPTION
**Pull request by AnthonyLaw**
## Changed
- footer content #183 
- Activity Buckets text #235

### CODE REVIEW

**Comment by Alexhuszagh** [comment]
Looks good to me.

File: src/components/NemCopyright.vue [hunk 1]
```diff
@@ -20,16 +20,24 @@
...
-          </li>
-        </ul>
+        <span>NEM.io Foundation 2019</span>
+        <span>|</span>
+        <span>All Rights Reserved</span>
+        <span>|</span>
+        <span>
+          <a href="/terms">Terms</a> &
+          <a href="/privacy">Privacy</a>
+        </span>
```
Comment (Alexhuszagh) [inline src/components/NemCopyright.vue]: Is there any particular reason for changing `<router-link ...>...< /router-link>` to `<a href="...">...</a>`?

Comment (AnthonyLaw) [reply src/components/NemCopyright.vue]: because of the previous click issue #187 
we are change all to <a> tag

Comment (Alexhuszagh) [reply src/components/NemCopyright.vue]: @AnthonyLaw Ah fair, thanks.

**Comment by Alexhuszagh** [comment]
@AnthonyLaw Can you fix the CSS style with `npm run lint:css`?

**Pull request merged**